app/main.py

```diff
@@ -1,2 +1,70 @@
+import dataclasses
+
+from typing import Any, Hashable
+
+
+@dataclasses.dataclass
 class Dictionary:
-    pass
+    capacity: int = 8
+    load_factor: float = 0.75
+    hash_table_: list = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]
+
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]